FinSearchUnified/Tests/Helper/UrlBuilderTest.php

```diff
@@ -77,10 +105,50 @@ public function testSendOnlyUniqueUserIps($unfilteredIp, $expectedValue)
     }
 
     /**
+     * @dataProvider reverseProxyIpAddressProvider
+     *
+     * @param string $unfilteredIp
```

Comment:
Shouldn't this be of type `array`?

Reply:
@ihanli No its a comma separated string actually. I did get confused earlier that why do you want me to type hint the `string`, but I thought if you want for this case then its fine with me.